### ISSUE TITLE
Loading page for survey results

### ISSUE THREAD

**Issue by isidoravladic** [Xcov19/mhc-fe#37]

**Comment by Kazz24** [comment]
Created loading page for user survey results 

https://user-images.githubusercontent.com/114235095/204318945-2abec69c-5eba-4290-8b01-da92dc5ee483.mp4

**Comment by codecakes** [comment]
Can we add this to the mockup flow we have

**Comment by Kazz24** [comment]
Yes, it's ready to be added to the mockup flow.  I think we're still waiting on which design set for the survey questions will be ultimately chosen.

**Comment by codecakes** [comment]
> Yes, it's ready to be added to the mockup flow. I think we're still waiting on which design set for the survey questions will be ultimately chosen.

I've responded https://github.com/Xcov19/mhc-fe/issues/27#issuecomment-1356444649
